docs/tutorial/piezotracking.rst

```diff
@@ -0,0 +1,136 @@
+.. warning::
+    This is alpha functionality. While usable, this has not yet been tested in a large
+    number of different scenarios. The API may also still be subject to change.
+
+Piezo tracking
+==============
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+In this tutorial, we will determine a high frequency distance (piezo distance) between the beads from the piezo mirror position of trap 1 and the measured force data.
+We will also show how to use the same reference curve to apply baseline correction for force signals in post processing.
+
+Positional Calibration
+----------------------
+
+The first step is to perform a positional calibration to calibrate the trap position to the distance measured by the camera.
+
+To do this calibration, we require a dataset acquired in the absence of a tether.
+In this dataset, trap 1 is moved over range, while the trap 1 position and camera-based distance is recorded.
+
+Note that this dataset should be acquired such that the bead tracking templates do not overlap.
+In case of overlap, make sure to slice the position and distance data such that this region is not included.
+Let's load this dataset and perform the distance calibration by invoking::
+
+    no_tether_data = lk.File("no_tether.h5")
+
+    distance_calibration = lk.DistanceCalibration(
+        calibration_data["Trap position"]["1X"], calibration_data.distance1, degree=2
+    )
+
+What this class does is regress a polynomial between the trap position and distance determined by the bead tracking.
+Note that this bead-to-bead distance typically already has the bead diameter subtracted, therefore it reflects the surface-to-surface distance.
+We can plot what this curve looks like by invoking `.plot()` on it::
```

Comment:
Also `"typically"` seems weird here. Is there actually the possibility to have data without the radius subtracted? If so, do we have a way to detect this or any other kind of mitigation strategy?

```suggestion
This class performs a polynomial regression between the trap position and  bead tracking distance. Note that this bead-to-bead distance typically already has the bead radius subtracted, therefore it reflects the surface-to-surface distance.
We can plot the calibration curve by invoking `.plot()` on it::
```

Reply:
Dropped typically indeed.